src/app-modules/docker-compose

```diff
@@ -21,8 +21,12 @@ if test -f "$MODULE_CONFIG_FILE"; then
     . "$MODULE_CONFIG_FILE"
 fi
 
-docker_compose_cmd=${DOCKER_COMPOSE_COMMAND:-"docker-compose"}
 docker_cmd=${DOCKER_COMMAND:-"docker"}
+docker_compose_cmd="${docker_cmd} compose"
+if [ -n "$USE_LEGACY_DOCKER_COMPOSE" ]; then
+  docker_compose_cmd="docker-compose"
+fi
```

Comment:
I do not like introducing another flag. maybe if `DOCKER_COMPOSE_COMMAND` is set, lets just assign to `docker_compose_cmd` and otherwise use `"${docker_cmd} compose"` (pretty much as you suggest, but without extra variable)

Reply:
we also need an explicit test case for both cases.

Reply:
I agree. I created another PR which discovers whether Docker compose is installed in standalone mode or as a plugin when invoked.